src/alert/alert.spec.ts

```diff
@@ -25,33 +30,40 @@ describe('ngb-alert', () => {
              expect(alertEl.getAttribute('role')).toEqual('alert');
              expect(getCloseButton(alertEl)).toBeTruthy();
            });
-     })));
+     }));
 
-  it('should allow specifying alert type', async(inject([TestComponentBuilder], (tcb) => {
-       tcb.overrideTemplate(TestComponent, '<ngb-alert type="success">Cool!</ngb-alert>')
-           .createAsync(TestComponent)
-           .then((fixture) => {
+  it('should allow specifying alert type', async(() => {
+       TestBed.overrideComponent(TestComponent, {set: {template: '<ngb-alert type="success">Cool!</ngb-alert>'}})
+           .compileComponents()
+           .then(() => {
+             const fixture = TestBed.createComponent(TestComponent);
              fixture.detectChanges();
              expect(getAlertElement(fixture.nativeElement)).toHaveCssClass('alert-success');
            });
-     })));
-
-  it('should render close button only if dismissible', async(inject([TestComponentBuilder], (tcb) => {
-       tcb.overrideTemplate(TestComponent, '<ngb-alert [dismissible]="false">Don\'t close!</ngb-alert>')
-           .createAsync(TestComponent)
-           .then((fixture) => {
+     }));
+
+  it('should render close button only if dismissible', async(() => {
+       TestBed
+           .overrideComponent(
+               TestComponent, {set: {template: '<ngb-alert [dismissible]="false">Don\'t close!</ngb-alert>'}})
```

Comment:
This should use a template string, so that the escaping of the single quote is unnecessary.